src/main/java/org/hvdw/jexiftoolgui/datetime/DateTime.java

```diff
@@ -33,19 +37,18 @@ public void setFileDateTimeToDateTimeOriginal( JProgressBar progressBar) {
                 // The < or > redirect options cannot directly be used within a single param on unixes/linuxes
                 cmdparams.add("/bin/sh");
                 cmdparams.add("-c");
-                tmpcmpstring = Utils.platformExiftool() + " -overwrite_original '-FileModifyDate<DateTimeOriginal' ";
+                tmpcmpstring = new StringBuilder(Utils.platformExiftool() + " -overwrite_original '-FileModifyDate<DateTimeOriginal' ");
             }
             for (int index: selectedIndices) {
-                //System.out.println("index: " + index + "  image path:" + files[index].getPath());
+                logger.trace("index: {} image path: {}", index, files[index].getPath());
                 if (isWindows) {
                     cmdparams.add(files[index].getPath().replace("\\", "/"));
                 } else {
-                    //cmdparams.add(files[index].getPath());
-                    tmpcmpstring += " " + files[index].getPath();
```

Comment:
On a loop, this is an expensive call. That is why I changed it to StringBuilder.